.github/workflows/changelog.yml

```diff
@@ -16,4 +16,5 @@ jobs:
     - name: Grep CHANGES.md for PR number
       if: contains(github.event.pull_request.labels.*.name, 'skip news') != true
       run: |
-        grep -P "PR #${{ github.event.pull_request.number }}[^0-9]" CHANGES.md
+        grep -P "PR #${{ github.event.pull_request.number }}[^0-9]" CHANGES.md || \
+        echo "Please add 'PR #${{ github.event.pull_request.number }}' change line to CHANGES.md"
```

Comment:
We might need a `; exit 1` after the echo so the job fails

Reply:
For future reference, a `; exit 1` was necessary to fail the job after the echo.